src/utilities/sequence-editor/sequence-linter.ts

```diff
@@ -302,141 +304,163 @@ function validateCommandTypeMixing(node: SyntaxNode): Diagnostic[] {
   return diagnostics;
 }
 
-function validateLocals(locals: SyntaxNode[], text: string) {
+function validateVariables(inputParams: SyntaxNode[], text: string, type: 'INPUT_PARAMS' | 'LOCALS' = 'LOCALS') {
   const variables: VariableDeclaration[] = [];
   const diagnostics: Diagnostic[] = [];
+
+  if (inputParams.length === 0) {
+    return {
+      diagnostics,
+      variables,
+    };
+  }
+
   diagnostics.push(
-    ...locals.slice(1).map(
-      local =>
+    ...inputParams.slice(1).map(
+      inputParam =>
         ({
-          ...getFromAndTo([local]),
-          message: 'There is a maximum of one @LOCALS directive per sequence',
+          ...getFromAndTo([inputParam]),
+          message: `There is a maximum of ${type} directive per sequence`,
```

Comment:
The new wording drops what the maximum allowed number is

Reply:
Addressed with commit 3244803be50f2ee2de696c1459c4dc03d3d4bb87